src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverted = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (inverted[value] !== undefined) {
+      // if a value is repeated, return null
+      return null;
+    }
+    inverted[value] = key;
```

Comment:
It's better to do it vice versa:
```
if (inverted[value] === undefined) {
  inverted[value] = key;
}
```
So you can avoid the additional return statement.